UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs

```diff
@@ -142,14 +142,52 @@ public GLTFRoot GetRoot()
 		public void SaveGLB(string path, string fileName)
 		{
 			_shouldUseInternalBufferForImages = true;
+			string fullPath = Path.Combine(path, Path.ChangeExtension(fileName, "glb"));
+			
+			using (FileStream glbFile = new FileStream(fullPath, FileMode.Create))
+			{
+				GLBToStream(glbFile, fileName);
+			}
+
+			if (!_shouldUseInternalBufferForImages)
+			{
+				ExportImages(path);
+			}
+		}
+
+		/// <summary>
+		/// In-memory GLB creation helper. Useful for platforms where no filesystem is available (e.g. WebGL).
+		/// </summary>
+		/// <param name="sceneName"></param>
+		/// <returns></returns>
+		public byte[] GetGLBData(string sceneName)
+		{
+			byte[] buf = null;
```

Comment:
There's no need for this variable.  You can just return the result of streamToArray() below.

```suggestion
```